File: 7KYU/Unlucky Days/solution.go
package kata

import "time"

func UnluckyDays(year int) int {
	  date := time.Date(year, time.January, 1, 0, 0, 0, 0, time.UTC)
    unlucky := 0
  
    for {
       if  date.Year() != year {
         break
       }
      
       if date.Day() == 13 && date.Weekday() == time.Friday {
         unlucky +=1
       }
       date = date.Add(time.Hour * 24)
    }
  
    return unlucky
}

File: 7KYU/Unlucky Days/solution.cs
using System;

public class Kata
{
  public static int UnluckyDays(int year)
  {
    DateTime date = new DateTime(year, 1, 1);
    int unluckyDays = 0;
    while(date.Year == year)
    {
        if(date.DayOfWeek == DayOfWeek.Friday && date.Day == 13)
          unluckyDays ++;
        date = date.AddDays(1);
    }
    return unluckyDays;
  }
}